crates/sui/src/ptb/ptb.rs

```diff
@@ -200,29 +207,31 @@ impl PTB {
     /// next command
     fn resolve_file(
         &self,
-        parent_file: &Option<String>,
+        cwd: PathBuf,
         filename: Vec<String>,
-        included_files: &mut BTreeMap<String, Vec<String>>,
-        current_file: String,
+        included_files: &mut BTreeMap<PathBuf, Vec<PathBuf>>,
+        current_file: PathBuf,
         start_index: usize,
         output: &mut BTreeMap<usize, PTBCommand>,
     ) -> Result<usize, anyhow::Error> {
         if filename.len() != 1 {
             return Err(anyhow!("The --file options should only pass one filename"));
         }
         let filename = filename.get(0).unwrap();
-        let file_path = std::path::Path::new(filename);
+        let file_path = std::path::Path::new(&cwd).join(filename);
+        let file_path = std::fs::canonicalize(file_path)
```

Comment:
This is something I am not 100% sure of. Particularly, dealing with symlinks is problematic, in the sense of, it will error on a file that does not exist. Which, in our case is what we want, but would appreciate another pair of eyes. 

Basically, if we have a file inclusion like this `--file ../test.ptb` we need to get the absolute path, and this `canonicalize` will do that for us.

Reply:
I think for now, not supporting symlinks is fine (I believe the package system will already have issues with that today). In the future we can add support for symlinks (along with other places) but I think that will be a bigger lift, and we should probably use a file-path library like https://docs.rs/path_abs/latest/path_abs/ or similar.